packages/contact-center/user-state/src/constants.ts

```diff
@@ -0,0 +1 @@
+export const AGENT_STATE_CHANGE = 'agent:stateChange';
```

Comment:
question. Do we expect to add more events in constants for the future? The reason I ask this is that we could have an enum of events.
Also, why don't we import the event from plugin-cc?

Reply:
I believe, no. At least for now, the SDK doesn't emit any more events. 

I agree we could import this from the SDK. The SDK as of now doesn't export this constant. I have added a TODO here to work on it. Will do so as part of a later PR